components/voucher/RedemptionList.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import DataTable from 'primevue/datatable'
+import Column from 'primevue/column'
+import ConfirmDialog from 'primevue/confirmdialog'
+import Button from 'primevue/button'
+import Card from 'primevue/card'
+import Skeleton from 'primevue/skeleton'
+import Toast from 'primevue/toast'
+import { useConfirm } from 'primevue/useconfirm'
+import { useToast } from 'primevue/usetoast'
+
+const route = useRoute()
+const router = useRouter()
+const toast = useToast()
+const confirm = useConfirm()
+
+const { data: redemptions, isLoading: redemptionsIsLoading, error: redemptionsError } = useClaimRedeems()
+const { mutate: updateMutate, isLoading: isUpdateLoading } = useAdminClaimRedeem()
+
+function claimVoucher(redemptionId: any, userId: any) {
+  updateMutate({
+    redemptionId,
+    userId,
+  },
+  {
+    onSuccess() {
+      toast.add({
+        summary: 'Voucher Claimed!',
```

Comment:
```suggestion
        summary: 'Voucher claimed!',
```

Reply:
My personal preference if I was an end user is to capitalize this I have looked at the UI with the the non capitalization of the  c in claimed